debugger/debugger.ts

```diff
@@ -814,6 +847,10 @@ export namespace Debugger {
 
     export function triggerBreak() {
         breakAtDepth = math.huge;
+        breakInThread = undefined;
+        if (hookStack.length > 0) {
```

Comment:
If `triggerBreak` is called before `debugFunction`, `runHook` will be set the debugger won't break immediately as expected.